frontend/src/pages/LegalPage.css

```
@@ -0,0 +1,67 @@
+}
+
+
+.legal-page__card {
+    max-width: 700px;
+    width: 100%;
+    text-align: left;
+}
+
+
+.legal-page__title {
+  font-family: 'Geologica', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geologica" (font-family-name-quotes)

(font-family-name-quotes)